tests/test_cli.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of cernopendata-client.
+#
+# Copyright (C) 2020 CERN.
+#
+# cernopendata-client is free software; you can redistribute it and/or modify
+# it under the terms of the GPLv3 license; see LICENSE file for more details.
+
+"""cernopendata-client text user interface(tui) test."""
+
+from click.testing import CliRunner
+from cernopendata_client.cli import get_record, get_file_locations, download_files
+
+
+def test_get_record():
+    """Test get_record() command."""
+    test_get_record = CliRunner()
+    test_result = test_get_record.invoke(get_record, ["--recid", 3005])
+    assert type(test_result.output) == str
```

Comment:
Instead of just asserting type string, you could also assert exit code (should be 0) and, more importantly, the presence of some expected text in the output.

Example:

```python
    assert '"title": "Configuration file for LHE step HIG-Summer11pLHE-00114_1_cfg.py"' in test_result.output
    assert test_result.exit_code == 0
```

The same for the other commands, e.g. when listing file locations, assert presence of a known one what the command should return, etc.